### ISSUE TITLE
LoadType.PREPEND issue 

### ISSUE THREAD

**Issue by DidarSeyidov** [BreakZero/EasyWallet-Multiplatform#39]
Does PagingCollectionViewController class support for paginating previous pages. It is necessary when set initial offset other than 0. Moreover when DB updated it reloads the latest page and I cannot reload previous pages by scrolling up

**Comment by BreakZero** [comment]
Yes it does, But you should make sure DB updated must trigger flow emit new paging data

**Comment by DidarSeyidov** [comment]
How can I trigger flow emit for a specific page. It reloads all beginning from first page

**Comment by BreakZero** [comment]
If it works on the Android side, that means you're using a hot flow. It will automatically emit data when db changes occur, then allowing the ViewModel to receive the new data. Can you debug the data of this workflow step by step?

**Comment by DidarSeyidov** [comment]
@NativeCoroutines
    fun getAllBooks(): Flow<PagingData<Books>> {
        
        return Pager(
            config = PagingConfig(pageSize = 30),
            pagingSourceFactory = {
                dataSource.getAllBooks()
            }
        ).flow.cachedIn(CoroutineScope(Dispatchers.IO))
    }
this is repository and called in viewModel of swiftUI. As you see it is cold flow

**Comment by DidarSeyidov** [comment]
fun getBooks() = QueryPagingSource(
        countQuery = database.booksQueryQueries.countBooks(),
        transacter = database.booksQueryQueries,
        context = Dispatchers.IO,
        queryProvider = { limit, offset ->
            database.booksQueryQueries.selectBooks(offset = offset, limit = limit)
        },
    )
this is dataSource, I cannot solve the issue please help
